collector/container/rhtap-slim.Dockerfile

```diff
@@ -0,0 +1,152 @@
+ARG BUILD_DIR=/build
+ARG CMAKE_BUILD_DIR=${BUILD_DIR}/cmake-build
+
+# TODO(ROX-20312): we can't pin image tag or digest because currently there's no mechanism to auto-update that.
+# TODO(ROX-20651): Use RHEL/ubi base image when entitlement is solved.
+# RPMs requiring entitlement: bpftool, cmake-3.18.2-9.el8, elfutils-libelf-devel, tbb-devel, c-ares-devel, jq-devel
+# FROM registry.access.redhat.com/ubi8/ubi:latest AS builder
+FROM quay.io/centos/centos:stream8 AS builder
+
+COPY . .
+
+# TODO(ROX-20234): use hermetic builds when installing/updating RPMs becomes hermetic.
+RUN dnf upgrade -y --nobest \
+    && dnf -y install --nobest \
+        autoconf \
+        automake \
+        binutils-devel \
+        bison \
+        ca-certificates \
+        clang-15.0.7 \
+        cmake \
+        cracklib-dicts \
+        diffutils \
+        elfutils-libelf-devel \
+        file \
+        flex \
+        gcc \
+        gcc-c++ \
+        gdb \
+        gettext \
+        git \
+        glibc-devel \
+        libasan \
+        libubsan \
+        libcap-ng-devel \
+        libcurl-devel \
+        libtool \
+        libuuid-devel \
+        make \
+        openssh-server \
+        openssl-devel \
+        patchutils \
+        passwd \
+        pkgconfig \
+        rsync \
+        tar \
+        unzip \
+        valgrind \
+        wget \
+        which \
+        bpftool \
+    && dnf clean all
+
+ARG BUILD_DIR
+ARG SRC_ROOT_DIR=${BUILD_DIR}
+ARG CMAKE_BUILD_DIR
+# TODO(ROX-20240): CMAKE_BUILD_TYPE should probably not be Release for PR, normal branch builds
+ARG CMAKE_BUILD_TYPE=Release
+# Appends an argument to the driver download URL that is used for filtering alerts on missing kernels.
```

Comment:
I'm not entirely sure this alerts exists anymore, would probably makes sense to verify.